package.json

```diff
@@ -32,14 +32,6 @@
     "helpers/**/*.js",
     "helpers/**/*.d.ts"
   ],
-	"bundlewatch": {
```

Comment:
I think we also need to remove the bundlewatch dependency & regenerate the lockfile